spec/services/distribution_create_service_spec.rb

```diff
@@ -21,8 +23,11 @@
       it "Sends a PartnerMailer" do
         @partner.update!(send_reminders: true)
 
-        expect(PartnerMailerJob).to receive(:perform_later).once
```

Comment:
If I keep this line in, `perform_enqueued_jobs` no longer works... which is weird...

Reply:
Yeh thats werid! I'am not entirely sure why either.